x/gov/contents.go

```diff
@@ -0,0 +1,76 @@
+package gov
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/gov/proposal"
+)
+
+//nolint
+const (
+	ProposalTypeText            string = "Text"
+	ProposalTypeSoftwareUpgrade string = "SoftwareUpgrade"
```

Comment:
I omitted because 1. there is no parameter change proposal now and 2. it is hard to migrate the existing format of parameter change proposal(which is basically just a textproposal) to the current format(which actually describes which parameter will be modified). Do you think we should migrate it to a textproposal?